scripts/stablex/place_margin_orders.js

```diff
@@ -0,0 +1,91 @@
+const BatchExchange = artifacts.require("BatchExchange")
+const { sendTxAndGetReturnValue } = require("../../test/utilities.js")
+const BN = require("bn.js")
+
+const argv = require("yargs")
+  .option("tokens", {
+    describe: "Collection of trusted tokens",
```

Comment:
Can we make the types explicit and how they are enumerated (e.g. with commas)

Reply:
Yes. I hope so. I will look into it.  Unless you just mean describe how they should be used in the description